scss/style.scss

```diff
@@ -2,5 +2,145 @@
 
 @import "variables";
 @import "typography";
-
 // import components here
+@font-face {
+    font-family: Raleway; 
+    src: url(src/fonts/Raleway-SemiBold.ttf); 
+   }
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Raleway, sans-serif;
+}
+
+a {
+    text-decoration: none;
+    color:#FAFAFA;
+}
+body {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+.start-screen-block {
+  width: $container-width;
+  height: 900px;
```

Comment:
Fixed height is veeery bad practice.

Reply:
I agree, but i think it's only way to do right according pixelperfect

Reply:
You can use indents for this situation (paddings, margins)